### PR TITLE
Added schemas for currency_conversion_config schema

### PR DESCRIPTION
**Pull request by AALEKH**

### CODE REVIEW

File: schemas/com.snowplowanalytics.snowplow/currency_conversion/jsonschema/1-0-0 [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+		"enabled": {
+			"type": "boolean"
+		},
+		"properties": {
+				"apiKey": {
+					"type": "string"
+				},
+				"baseCurrency": {
+					"type": "string"
+				}
```
Comment (alexanderdean) [inline schemas/com.snowplowanalytics.snowplow/currency_conversion/jsonschema/1-0-0]: Can you make both fields required and also add additionalProperties

**Comment by alexanderdean** [comment]
Can you rename currency_conversion to currency_conversion_config?

**Comment by AALEKH** [comment]
Sure, Will be done

File: schemas/com.snowplowanalytics.snowplow/currency_conversion_config/jsonschema/1-0-0 [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+				            "baseCurrency": {
+					                  "type": "string"
+				            },
+				            "rateAt": {
+				                "enum": ["EOD_PRIOR"]
+				            }
+				    },         
+			        "required": ["apiKey", "baseCurrency", "rateAt"],
+			        "additionalProperties": false       
+		    }
```
Comment (alexanderdean) [inline schemas/com.snowplowanalytics.snowplow/currency_conversion_config/jsonschema/1-0-0]: Can you clean up the indentation in this one please @AALEKH? It's a little out of whack

Comment (AALEKH) [inline schemas/com.snowplowanalytics.snowplow/currency_conversion_config/jsonschema/1-0-0]: Sure !!

**Comment by AALEKH** [comment]
@alexanderdean Done !!

File: schemas/com.snowplowanalytics.snowplow/currency_conversion_config/jsonschema/1-0-0 [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+				"baseCurrency": {
+			        "type": "string"
+				},
+				"rateAt": {
+				    "enum": ["EOD_PRIOR"]
+				}
+		    },         
+			"required": ["apiKey", "baseCurrency", "rateAt"],
+			"additionalProperties": false       
+		}
```
Comment (alexanderdean) [inline schemas/com.snowplowanalytics.snowplow/currency_conversion_config/jsonschema/1-0-0]: Whitespace still looks wrong @AALEKH

Comment (alexanderdean) [inline schemas/com.snowplowanalytics.snowplow/currency_conversion_config/jsonschema/1-0-0]: Run it through a JSON prettifier

Comment (AALEKH) [inline schemas/com.snowplowanalytics.snowplow/currency_conversion_config/jsonschema/1-0-0]: Ok, I will fix this, Thanks !!

Comment (alexanderdean) [inline schemas/com.snowplowanalytics.snowplow/currency_conversion_config/jsonschema/1-0-0]: Can you fix this ASAP @AALEKH - I want to do an Iglu Central release this afternoon...

Comment (AALEKH) [inline schemas/com.snowplowanalytics.snowplow/currency_conversion_config/jsonschema/1-0-0]: Sure!! On it

**Comment by AALEKH** [comment]
@alexanderdean

**Comment by alexanderdean** [comment]
Looks good, thanks

**Comment by alexanderdean** [comment]
This is done in release 27, closing...

**Pull request closed without merging**